week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -37,9 +45,21 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(arr) {
-  
-}
+function formatPercentage(arr){
+function mystr(strarr) {
```

Comment:
This works well, well done! One thing worth thinking about a bit is naming here.
`mystr` isn't the most descriptive name - perhaps this could be called `formatOnePercentage` or something similar?
`strarr` is actually a little misleading as a parameter name, as are `Arr` and `newArr` for variable names, as they suggest that the variables are arrays when they're not. Some more descriptive names may be `number`, `numberString`, and `formattedPercentage`, or something along those lines... Something to think about :)